ntbs-ui-tests/testSettings.json

```diff
@@ -7,6 +7,10 @@
     "LeedsServiceUser":  {
       "Username": "kamala@aptemus.com",
       "TbServiceCode": "TBS0106"
+    },
+    "ReadOnlyUser":  {
+      "Username": "read_only@aptemus.com",
```

Comment:
As discussed earlier this username needs to be `readonly-uitester@aptemus.com` to be consistent with the new users that are being added in NTBS-2379 #1301